lib/morpheus.gi

```diff
@@ -1978,6 +1978,25 @@ local A;
   return A;
 end);
 
+#############################################################################
+##
+#M  AutomorphismGroup(<G>) . . abelian case
+##
+InstallMethod(AutomorphismGroup,"test abelian",true,[IsGroup and IsFinite],
```

Comment:
I've tested this PR on Jenkins, and tests pass. A minor remark is that the method's description `"test abelian"` does not seem to be enough informative. `"for a finite group to enforce check of IsAbelian"` maybe?